### ISSUE TITLE
Filtering with integer values for a string field will break in PostGres. 

### ISSUE THREAD

**Issue by vivekg342** [dchester/epilogue#130]
Postgres will fail with an error "operator does not exist: character varying = integer" when trying to filter with an integer value for a string field.
https://github.com/dchester/epilogue/blob/master/lib/Controllers/list.js#L18

For example
- JSON.parse("1234")  will return 1234. 
- Even though this field is declared as "STRING" in model definition.

Currently I had to make it work with an ugly hack

```
epilogue.Controllers.list.prototype._safeishParse = function(value) {
 return value;
};
```

**Comment by mbroadst** [comment]
@vivekg342 Hey, this should be relatively easy by replacing `safeishParse` with something like the following:

```
List.prototype._safeishParse = function(value, type) {
  var types = this.resource.sequelize;
  if (type instanceof types.STRING) { // do something for strings }
  if (type instanceof types.NUMBER) { // do something for numbers }

  try {
    return JSON.parse(value);
  } catch(err) {
    return value;
  }
};
```

and the calling context to:

```
  // all other query parameters are passed to search
  var extraSearchCriteria = _.reduce(req.query, function(result, value, key) {
    if (_.has(model.rawAttributes, key)) result[key] = self._safeishParse(value, model.rawAttributes[key].type);
    return result;
  }, {});
```

Unfortunately I have limited time to work on this project, but PRs will be graciously accepted. That should be the gist of the work, it just needs to be completed and tests should be added.

Thanks!

**Comment by mbroadst** [comment]
fixed in aaac7b6
